src/components/filters/MeasureValueFilter/OperatorDropdownItem.tsx

```diff
@@ -48,6 +53,19 @@ export class OperatorDropdownItem extends React.PureComponent<IOperatorDropdownI
         onClick(operator);
         e.preventDefault();
     };
+
+    private renderBubble(message: string) {
```

Comment:
You can pick `bubbleText` property from `props` here, not needed to pass it from `render` method.